README.rst

```diff
@@ -1,4 +1,4 @@
-Kebechet
+Kebechet 
```

Comment:
You can use `recheck` as a comment to trigger CI.

Reply:
Okay, noted :flushed: oops